selfdrive/car/toyota/interface.py

```diff
@@ -147,7 +147,6 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
       tune.kiV = [0.5]
       ret.vEgoStopping = 0.25
       ret.vEgoStarting = 0.25
-      ret.stoppingDecelRate = 0.3  # reach stopping target smoothly
```

Comment:
This current value already is a bit jerky on my hybrid, is it better on our ICE cars?